bot/kodiak/evaluation.py

```diff
@@ -928,6 +928,8 @@ async def set_status(msg: str, markdown_content: Optional[str] = None) -> None:
                 )
             )
             await api.create_comment(body)
+        else:
+            await set_status("merge complete 🎉")
```

Comment:
I'm not sure it matters, but if an existing PR is already merged we will not set this status message. Maybe that's a good thing because that's fewer API calls wasted.